### ISSUE TITLE
Can't scroll on encryptr.org

### ISSUE THREAD

**Issue by jdm** [servo/servo#6859]
Visit http://encryptr.org and attempt to scroll vertically. You can't.

**Comment by HarryLovesCode** [comment]
I wrote a very simple case that exhibits the issue you are talking about:

```html
<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <title>Test</title>
    </head>

    <style>
        html, body { height: 100vh; }
        .test0, .test1 { height: 100vh; }
        .test0 { background: white; }
        .test1 { background: purple; }
    </style>
    <body>
        <div class="test0"></div>
        <div class="test1"></div>
    </body>
</html>
```

This code should produce a white `div` that takes up the whole screen. When you scroll, the purple `div` should come into view. This expected behaviors does not occur with servo, but does with Firefox, Chrome, and Safari.

The relevant line is the one with `html, body {}`. Both 

- `html, body { height: 100vh; }`
- `html, body { height: 100%; }`

cause the issue. However, this is just one of the CSS snippets that can prevent layer scrolling. I'm currently looking for another which affects websites that don't have the code I included above. Also, here are some more websites I've encountered with the issue:

- [Roon Ghost Theme](https://roon.ghost.io)
- [Datafox Blog](http://eng.datafox.co)
- [Atom Editor Homepage](https://atom.io/)
- [Atom Editor Packages](https://atom.io/packages)
